package_build_list_host_linux.json

```diff
@@ -71,7 +72,8 @@
         "DirectXShaderCompilerDxc-1.6.2112-o3de-rev1-linux": "package-system/DirectXShaderCompilerDxc-linux",
         "azslc-1.8.15-rev2-linux": "package-system/azslc/temp/azslc-linux",
         "tiff-4.2.0.15-rev3-linux": "package-system/tiff-linux",
-        "python-3.10.5-rev2-linux": "package-system/python/linux_x64/package",
+        "python-3.10.5-rev3-linux": "package-system/python/temp/python-linux",
+        "python-3.10.5-rev3-linux-aarch64": "package-system/python/temp/python-linux-aarch64",
```

Comment:
Since the `package_build_list_host_*.json` are merged into when running the build_package.py script from the [3p-package-scripts](https://github.com/o3de/3p-package-scripts/blob/main/o3de_package_scripts/common.py#L394-L413) repo, the ArchLinux packages only need to be specified in the `package_build_host_linux-aarch64.json` file and not here.